cmake/BLTOptions.cmake

```diff
@@ -77,3 +77,17 @@ if( NOT ( ( BLT_CXX_STD STREQUAL "c++98" )
     message(FATAL_ERROR "${BLT_CXX_STD} is an invalid entry for BLT_CXX_STD.
 Valid Options are ( c++98, c++11, c++14 )")
 endif()
+
+
+################################
+# Advanced configuration options
+################################
+
+option(ENABLE_GTEST_DEATH_TESTS
```

Comment:
I named the option ``ENABLE_GTEST_DEATH_TESTS`` and marked it as advanced.  
For consistency with the other options, I did not prefix it with ``BLT``.

Does that work for everyone?